assets/sass/modules/subscribe-with-google/single-posts.scss

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
See above, let's deal with the WordPress editor and frontend integration (also for CSS) via a separate PR.